streams/src/main/java/org/apache/kafka/streams/errors/internals/DefaultErrorHandlerContext.java

```diff
@@ -81,6 +81,17 @@ public TaskId taskId() {
         return taskId;
     }
 
+    @Override
+    public String toString() {
+        return "ErrorHandlerContext{" +
+            "topic='" + topic + '\'' +
+            ", partition=" + partition +
+            ", offset=" + offset +
+            ", processorNodeId='" + processorNodeId + '\'' +
+            ", taskId=" + taskId +
+            '}';
+    }
```

Comment:
Omitting `headers` on purpose, as we should not log user-data.

Reply:
Good question. Sounds like a miss... \cc @sebastienviale @loicgreffier

Reply:
Side comment: I would not block this PR on this question, but do a follow up one, if we want to add ts.

Reply:
Yep I agree it shouldn't block, adding a timestamp in a follow on PR is fine.